events/wc_court_events.txt

```diff
@@ -160,7 +160,12 @@ character_event = {
 		picture = GFX_evt_nraqi
 	}
```

Comment:
If they reproduce sexually, where they are going to take the second ancient? Present Debutante invites ankoan female.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108753163-cefc9480-755d-11eb-99ef-d491bae2ff0c.png)

</details>